### ISSUE TITLE
allow block after => in function definition expressions

### ISSUE THREAD

**Issue by martin-henz** [source-academy/js-slang#87]
Many examples are much easier this way.

**Comment by openorclose** [comment]
Should the arrow function syntax be extended instead, allowing 

```javascript
function test(){
  return () => {
    multiple;
    return expressions;
  };
}
```

For students, this might be easier to understand as function declarations and arrow functions have vastly different syntax. If function expressions are used, students would have to understand why
```javascript
function f() {
  return 1;
}
```
and 
```javascript
(function f() {
  return 1;
});
```

mean completely different things but their syntax can be identical at times.

**Comment by martin-henz** [comment]
Yes, good idea!

**Comment by thomastanck** [comment]
Closed by #116
